ui/src/virtualconsole/vccuelist.cpp

```diff
@@ -817,25 +817,32 @@ void VCCueList::slotCurrentStepChanged(int stepNumber)
 
         float stepVal;
         int stepsCount = m_tree->topLevelItemCount();
-        if (stepsCount < 256)
-            stepVal = 255.0 / (float)stepsCount;
-        else
+        if (stepsCount < 256) {
```

Comment:
Please stick to the style of the rest of the code

Reply:
Oops - and this after you just checked in a huge lot of code style fixes barely a week ago. Sorry about that. Fixed, and pushed to this branch just now.  Awaiting checks but hopefully they'll complete just fine in a bit.